wrangler-service/src/main/java/co/cask/wrangler/service/directive/DirectivesHandler.java

```diff
@@ -1166,7 +1092,7 @@ private static Row createUberRecord(List<Row> rows) {
    * @return records generated from the directives.
    */
   private List<Row> executeDirectives(NamespacedId id, @Nullable Request user,
-                                      Function<List<Row>, List<Row>> sample) {
+                                      Function<List<Row>, List<Row>> sample) throws RecipeException {
```

Comment:
Isn't `RecipeException` handled in line 1120?

Reply:
whoops, good catch.